.github/workflows/example-node-versions.yml

```diff
@@ -14,8 +14,9 @@ jobs:
     runs-on: ubuntu-22.04
     # let's make sure Cypress works on several versions of Node
     strategy:
+      fail-fast: false
```

Comment:
never did understand why GitHub Actions made this default to `true`... I usually don't want to skip other CI stages just because one failed :shrug:

Reply:
I guess it depends, but at least there is a choice!